### PR TITLE
Niamh25 doc 2893 note specifying only the stated third parties are supported

### PR DESCRIPTION
**Pull request by Niamh25**
Note specifying only the stated third parties are supported

### CODE REVIEW

File: content/en/platform/corda/4.10/enterprise/node/setup/host-prereq.md [hunk 1]
```diff
@@ -57,6 +57,9 @@ weight: 2
 
 {{< /table >}}
 
+{{< note >}}
+Third party managed service variants are not supported unless explicitly stated.
```
Comment (nadinequinr3) [inline content/en/platform/corda/4.10/enterprise/node/setup/host-prereq.md:61]:
```suggestion
Third-party managed service variants are not supported unless explicitly stated.
```

File: content/en/platform/corda/4.8/enterprise/node/setup/host-prereq.md [hunk 2]
```diff
@@ -57,6 +57,10 @@ weight: 2
 
 {{< /table >}}
 
+{{< note >}}
+Third party managed service variants are not supported unless explicitly stated.
```
Comment (nadinequinr3) [inline content/en/platform/corda/4.8/enterprise/node/setup/host-prereq.md:61]:
```suggestion
Third-party managed service variants are not supported unless explicitly stated.
```

File: content/en/platform/corda/4.9/enterprise/node/setup/host-prereq.md [hunk 3]
```diff
@@ -57,6 +57,9 @@ weight: 2
 
 {{< /table >}}
 
+{{< note >}}
+Third party managed service variants are not supported unless explicitly stated.
```
Comment (nadinequinr3) [inline content/en/platform/corda/4.9/enterprise/node/setup/host-prereq.md:61]:
```suggestion
Third-party managed service variants are not supported unless explicitly stated.
```

**Review by nadinequinr3 — CHANGES_REQUESTED** [review]

**Review by nadinequinr3 — APPROVED** [review]

**Review by yarivus — COMMENTED** [review]
thx @Niamh25

**Pull request merged**